src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -1,25 +1,40 @@
-import { Box, Container } from '@mui/material';
-import { ImageProps } from 'next/image';
+import { Box, Container, SxProps, Theme } from '@mui/material';
```

ESLint

[error] 'SxProps' is defined but never used.

(@typescript-eslint/no-unused-vars)